CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [5.2.0] - 2022-05-XX
```

Comment:
Remove this. On release (through maven release plugin), this will be added automaticly by https://github.com/enear/keepachangelog-maven-plugin

Reply:
Done